python/rpdk/java/templates/init/guided_aws/StubCreateHandler.java

```diff
@@ -0,0 +1,138 @@
+package {{ package_name }};
+
+// TODO: replace all usage of SdkClient with your service client type, e.g; YourServiceAsyncClient
+// import software.amazon.awssdk.services.yourservice.YourServiceAsyncClient;
+
+import java.util.Objects;
+import software.amazon.awssdk.awscore.AwsRequest;
+import software.amazon.awssdk.awscore.AwsResponse;
+import software.amazon.awssdk.awscore.exception.AwsServiceException;
+import software.amazon.awssdk.core.SdkClient;
+import software.amazon.cloudformation.exceptions.CfnAlreadyExistsException;
+import software.amazon.cloudformation.exceptions.CfnGeneralServiceException;
+import software.amazon.cloudformation.exceptions.CfnNotFoundException;
+import software.amazon.cloudformation.proxy.AmazonWebServicesClientProxy;
+import software.amazon.cloudformation.proxy.Logger;
+import software.amazon.cloudformation.proxy.ProgressEvent;
+import software.amazon.cloudformation.proxy.ProxyClient;
+import software.amazon.cloudformation.proxy.ResourceHandlerRequest;
+
+public class {{ operation }}Handler extends BaseHandlerStd {
+    private Logger logger;
+
+    protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(
+        final AmazonWebServicesClientProxy proxy,
+        final ResourceHandlerRequest<ResourceModel> request,
+        final CallbackContext callbackContext,
+        final ProxyClient<SdkClient> proxyClient,
+        final Logger logger) {
+
+        this.logger = logger;
+
+        final ResourceModel model = request.getDesiredResourceState();
+
+        // TODO: Adjust Progress Chain according to your implementation
+
+        return ProgressEvent.progress(model, callbackContext)
+
+            // STEP 1 [check if resource already exists]
+            // if target API does not support 'ResourceAlreadyExistsException' then following check is required
+            //.then(progress -> checkForPreCreateResourceExistence(proxy, request, progress))
+
+            // STEP 2 [create/stabilize progress chain - required for resource creation]
+            .then(progress ->
+                // If your service API throws 'ResourceAlreadyExistsException' for create requests then CreateHandler can return just proxy.initiate construction
+                // STEP 2.0 [initialize a proxy context]
+                proxy.initiate("{{ call_graph }}::{{ operation }}", proxyClient, model, callbackContext)
+
+                    // STEP 2.1 [TODO: construct a body of a request]
+                    .request(Translator::translateToCreateRequest)
+
+                    // STEP 2.2 [TODO: make an api call]
+                    .call(this::createResource)
+
+                    // STEP 2.3 [TODO: stabilize is describing the resource until it is in a certain status]
+                    .stabilize(this::stabilizedOnCreate)
```

Comment:
can we add optional postCreate as well? again, this is described in the handler contract

"""
Stabilization is the process of waiting for a resource to be in a particular state.  For Create and Update handlers, desired-state stabilization is satisfied when all properties specified in the request are applied as requested.  This is verified by calling the Read handler.  Note that reaching the desired-state is mandatory for all handlers before returning SUCCESS.

In many cases, the desired-state is reached immediately upon completion of the Create/Update API call.  However, in some cases, multiple API calls and/or wait periods may be required in order to reach this state.
"""